### ISSUE TITLE
EOF issue

### ISSUE THREAD

**Issue by paulianttila** [DerDanilo/proxmox-stuff#3]
I met following problem when tried to execute backup script in Proxmox VE 5.1
```
root@proxmox:~# ./prox_config_backup.sh
./prox_config_backup.sh: line 115: warning: here-document at line 28 delimited by end-of-file (wanted `EOF')
./prox_config_backup.sh: line 116: syntax error: unexpected end of file

```

I removed all space characters before EOF and script started to work fine.

```
function description {
    clear
    cat <<EOF
        Proxmox Server Config Backup
        Hostname: "$_HOSTNAME"
        Timestamp: "$_now"
        Files to be saved:
        "/etc/*, /var/lib/pve-cluster/*, /root/*"
        Backup target:
        "$_bdir"
        -----------------------------------------------------------------
        This script is supposed to backup your node config and not VM
        or LXC container data. To backup your instances please use the
        built in backup feature or a backup solution that runs within
        your instances.
        For questions or suggestions please contact me at
        https://github.com/DerDanilo/proxmox-stuff
        -----------------------------------------------------------------
        Hit return to proceed or CTRL-C to abort.
EOF
    read dummy
    clear
}
```

I read somewhere that tabs (not spaces) should work with indentation, but I didn't test it.

**Comment by DerDanilo** [comment]
Thanks for reporting. I don't have this problem but know that it exists depending on the backend. Are you using bash to execute the script?

**Comment by paulianttila** [comment]
Yes, bash (4.4.12(1)-release).

From bash man page:

Here Documents
**This type of redirection instructs the shell to read input from the current source until a line containing only delimiter (with no trailing blanks) is seen.**  All of  the  lines read up to that point are then used as the standard input (or file descriptor n if n is specified) for a command.

The format of here-documents is:

	[n]<<[-]word
		here-document
	delimiter

**Comment by DerDanilo** [comment]
Sorry, I forgot about this issue. I'll aproach it after my vacation. There are quiet a few changes to the script.

**Comment by paulianttila** [comment]
Just tested the latest version and it seems that Issue was fixed already by the PR #2
